matchbox_socket/Cargo.toml

```diff
@@ -56,3 +56,6 @@ async-compat = { version = "0.2.1", default-features = false }
 
 [dev-dependencies]
 futures-test = { version = "0.3" }
+
+[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
+tokio = "1.12"
```

Comment:
Does tokio have any advantage over futures-test, which we used for the existing test. (I don't really have any strong preference either way, just thinking we should stick to one if we can)

Reply:
Changed to futures-test